mlir/test/CAPI/llvm.c

```diff
@@ -296,8 +296,13 @@ static void testDebugInfoAttributes(MlirContext ctx) {
   mlirAttributeDump(mlirLLVMDILocalVariableAttrGet(ctx, compile_unit, foo, file,
                                                    1, 0, 8, di_type));
   // CHECK: #llvm.di_derived_type<{{.*}}>
+  // CHECK-NOT: dwarfAddressSpace
+  mlirAttributeDump(mlirLLVMDIDerivedTypeAttrGet(ctx, 0, bar, di_type, 64, 8, 0,
+                                                 -1, di_type));
```

Comment:
```suggestion
                                                 MLIR_CAPI_DWARF_ADDRESS_SPACE_NULL, di_type));
```
nit: Can we use the "constant" here?

Reply:
Thanks, I wrote the test before I decided to add the constant and forgot to circle back.